### PR TITLE
Add ${KUBE_RBAC_PROXY_IMAGE} replacer to CSI driver controller

### PR DESCRIPTION
**Pull request by jsafrane**
We use kube-rbac-proxy to get https endpoint for metrics.

cc @bertinatto

### CODE REVIEW

File: pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ const (
 	resizerImageEnvName       = "RESIZER_IMAGE"
 	snapshotterImageEnvName   = "SNAPSHOTTER_IMAGE"
 	livenessProbeImageEnvName = "LIVENESS_PROBE_IMAGE"
+	kubeRBACPRoxyImageEnvName = "KUBE_RBAC_PROXY_IMAGE"
```
Comment (bertinatto) [inline pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go:35]: nit: P**R**oxy

Comment (jsafrane) [reply pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go]: Fixed

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**